README.md

```diff
@@ -447,3 +447,184 @@ A **state category grade** represents the grade assigned to a state based on a s
 #### GET /grades/{code}
 
 Returns the state's overall grade, and its grades for each category. If no state with that code exists, it will return a 404 response code.
+
+### Forms
+
+A **form** represents any one of the forms found on the FreeFrom Map frontend. The forms include:
+
+* Give Feedback
+* Report Missing or Outdated Information
+* Partner with FreeFrom
+* Build Collective Survivor Power
+* Share your Policy Ideas
+
+These endpoints don't require authentication. Note that form submissions cannot be updated or overwritten.
+
+#### POST /forms/feedback
+
+This endpoint writes to the 'Give Feedback' sheet, and sends an email notification to the FreeFrom staff.
+
+All fields are optional, and values will be written to the sheet as plain text.
+
+**Request**
+
+| Name        | Type     | Question                | Notes       |
+|-------------|----------|-------------------------|-------------|
+| useful      | String   | "Was this tool useful?" | *Optional*. |
+| useful_desc | String   | "Can you tell us more about how the tool was or was not useful for you?" | *Optional*. |
+| learned     | String   | "Did you learn anything about policies related to survivor wealth from the tool?" | *Optional*. |
+| usage_plan  | [String] | "How do you plan to use this tool?" | *Optional*. |
+| suggestions | String   | "What can be improved or changed?" | *Optional*. |
```

Comment:
These params (same for the forms below) really come from the values in the spreadsheet, so we should make sure the spreadsheet is configured like [this template](https://docs.google.com/spreadsheets/d/1tKcaWGBS4z0_GY9F612T73NjUIUVoLP5oMLBkpNEOoU/edit?usp=sharing).